docs/cmdlets-reference/get-cohesityphysicalagentstatus.md

```diff
@@ -0,0 +1,40 @@
+# Get-CohesityPhysicalAgentStatus
```

Comment:
can't this be : Get-CohesityAgentStatus ?

Reply:
4 words seem too long for a cmdlet

Reply:
Renamed the cmdlet to Get-CohesityAgentStatus